README.md

```diff
@@ -455,6 +455,16 @@ response.card({
 });
 ```
 
+# Custom Directives
+
+The `response.directive(Object directive)` method allows you to set custom directive objects to devices to perform a specific device-level actions.
+
+The full specification for the `directive` object passed to this method can be found [here](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/alexa-skills-kit-interface-reference#response-format)
+
+`alexa-app` has special handling for AudioPlayer directives, so you only need to use this method for more general custom directives.
+
+`response.directive` adds your directive object to the directives array in the response. Currently, there is no way to remove a directive from the response once it is set except through `response.clear()`.
```

Comment:
Same as above ... 

I also think we could add the interface for removing a directive and return all directives. Maybe `response.directive` could return an object that has a `remove` method? And we could give access to directives via a `directives` collection as is?